sitemap_generate/generator.py

```diff
@@ -50,6 +50,7 @@ def record(self, url: str) -> bytes:
         environ = {
             'REQUEST_METHOD': 'GET',
             'wsgi.input': '',
+            'wsgi.url_scheme': defaults.SITEMAP_PROTO,
```

Comment:
According to [PEP-333](https://www.python.org/dev/peps/pep-0333/#environ-variables) also missing required variables:
* [ ] `SCRIPT_NAME` - may be empty string
* [ ] `SERVER_PROTOCOL` - `"HTTP/1.1"`
* [ ] `wsgi.version` - `(1, 0)`
* [ ] `wsgi.errors` - `StringIO()` - same type as for `wsgi.input`
* [ ] `wsgi.multithread` - `False`
* [ ] `wsgi.multiprocess` - `False`
* [ ] `wsgi.run_once` - `False`

Could you please add these and fix `wsgi.input` type?